test/functional/legacy/097_glob_path_spec.lua

```diff
@@ -28,47 +28,25 @@ describe('glob() and globpath()', function()
     -- Consistent sorting of file names
     command('set nofileignorecase')
 
-    if helpers.is_os('win') then
```

Comment:
This test is mostly the same as `Test_glob()` in `test_escaped_glob.vim`, which is skipped on non-Unix.